x-pack/plugins/maps/public/shared/layers/sources/kibana_tilemap_source/create_source_editor.js

```diff
@@ -21,20 +21,19 @@ const NO_TILEMAP_LAYER_MSG =
 
 export class CreateSourceEditor extends Component {
 
-
   state = {
     url: null
-  }
+  };
 
   _loadUrl = async () => {
     const tilemap = await getKibanaTileMap();
     if (this._isMounted) {
       this.setState({
         url: tilemap.url
       });
-      this.props.previewTilemap(this.props.url);
+      this.props.previewTilemap(this.state.url);
```

Comment:
You might want to perform this action as a second arg callback to setState to ensure you're getting the most recently set version of `this.state.url`. Having said that, I tested it locally and didn't have any issues.